playbooks/02 - Use Case - SOC Simulator/Reset Scenario.json

```diff
@@ -49,13 +49,14 @@
                     "item": "{{vars.demoRecordIRIs}}",
                     "__bulk": false,
                     "parallel": true,
-                    "condition": ""
+                    "condition": "{{'mitre' not in vars.item}}"
```

Comment:
make it a list in set variable and user here. so in future, we will need to add items in the list.